src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copyState = { ...state };
+  const massiveToReturn = [];
```

Comment:
[NAMING] - The variable `massiveToReturn` is not descriptive of its purpose. A more descriptive name such as `stateHistory` or `stateSnapshots` would be better.